docs/transpile/custom-transpiler-pass.ipynb

```diff
@@ -0,0 +1,239 @@
+      "text/plain": [
+       "<Figure size 327.397x204.68 with 1 Axes>"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "qc = QuantumCircuit(3)\n",
+    "qc.cx(0, 1)\n",
+    "qc.ecr(1, 2)\n",
+    "qc.ecr(1, 0)\n",
+    "qc.cx(2, 1)\n",
+    "qc.draw('mpl', style='iqp')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Since we are applying a single pass, we can use the shorthand for `PassManager(PauliTwirl()).run(qc)` to generate 50 randomized circuits. "
```

Comment:
I think it's still worth showing the longhand code snippet as well, otherwise it can feel a bit overwhelming if this is your first time working through this process.